From 9fd2850b33f8a6435ce66588b61bb7f1f2cd3606 Mon Sep 17 00:00:00 2001
From: Rene Engelhard <rene@debian.org>
Date: Sun, 10 May 2020 00:09:14 +0200
Subject: [PATCH] skia: add SkOpts_crc32

I get

/usr/bin/ld:
/home/rene/libreoffice-7.0.0~alpha1/workdir/GenCxxObject/UnpackedTarball/skia/src/core/SkOpts.o:
in function `SkOpts::Init()':
././workdir/UnpackedTarball/skia/src/core/SkOpts.cpp:135: undefined
reference to `SkOpts::Init_crc32()'
collect2: error: ld returned 1 exit status
make[2]: ***
[/home/rene/libreoffice-7.0.0~alpha1/external/skia/Library_skia.mk:10:
/home/rene/libreoffice-7.0.0~alpha1/instdir/program/libskialo.so] Error
1

on aarch64 otherwise

Change-Id: I713df3438e48fb7cb08ecdb07a998d21068ab464
---
 external/skia/Library_skia.mk | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/external/skia/Library_skia.mk b/external/skia/Library_skia.mk
index 6047de0b864b..96566b958a7c 100644
--- a/external/skia/Library_skia.mk
+++ b/external/skia/Library_skia.mk
@@ -774,6 +774,9 @@ $(eval $(call gb_Library_add_generated_exception_objects,skia,\
 $(eval $(call gb_Library_add_generated_exception_objects,skia,\
     UnpackedTarball/skia/src/opts/SkOpts_ssse3, $(CXXFLAGS_INTRINSICS_SSSE3) $(CLANG_CXXFLAGS_INTRINSICS_SSSE3) \
 ))
+$(eval $(call gb_Library_add_generated_exception_objects,skia,\
+    UnpackedTarball/skia/src/opts/SkOpts_crc32 \
+))
 
 $(eval $(call gb_Library_add_generated_exception_objects,skia,\
     UnpackedTarball/skia/tools/gpu/vk/VkTestUtils \
-- 
2.20.1

